Frontend/src/components/Layout/style.scss

```diff
@@ -3,26 +3,386 @@
 
 ///Global custom properties
 :root {
-    --spacing: 1rem;
-    --spacing-xs: calc(var(--spacing) * 0.25);
-    --spacing-sm: calc(var(--spacing) * 0.5);
-    --spacing-md: calc(var(--spacing) * 0.75);
-    --spacing-lg: calc(var(--spacing) * 3);
-    --spacing-xlg: calc(var(--spacing) * 5);
+    --spacing: 1rem;                             // 16px
+    --spacing-md: calc(var(--spacing) * 0.75);   // 12px
+    --spacing-lg: calc(var(--spacing) * 3);      // 48px
+
+    // Style guide spacing values
+    --spacing-xxs: calc(var(--spacing) * 0.25);   // 4px
```

Comment:
Are all these new properties used?

Reply:
I have not gone through the code to check that, but they are mentioned in the style guide, which means something probably uses/is supposed to use them

Reply:
I'm personally not a big fan of unused code. I'm not sure we're copying things from the design so closely we're also copying all their variables. I'd only copy the variables we're using. I'm not sure taking over the entire list is useful as-is. @arnoutj What do you think?

Reply:
It is evident that hard coded pixels creep into the code. I imagine that would happen less if there are already variables to use instead, but I am just guessing.

Reply:
I'm not opposed to merging it like this.. but lets wait for arnout's comments on this.

Reply:
I think it’s ok to have the variables defined even if unused. That’s better than hard coding pixels.

Reply:
Ok, if arnout agrees, let's merge it then.